src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery;
+        for (int i = 0; i < 3; i++) {
+            lottery = new Lottery();
+            System.out.println(lottery.getRandomBall());
+        }
         // create three balls using class Lottery and print information about them in console
```

Comment:
Please don't add redundant empty lines to your code. There is no need for an empty line after the loop before the closing brace.